packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -52,6 +53,17 @@ describe('RestServer (integration)', () => {
     expect(server.url).to.be.undefined();
   });
 
+  it('parses query without decorated rest query params', async () => {
+    // See https://github.com/strongloop/loopback-next/issues/2088
+    const server = await givenAServer({rest: {port: 0, host: '127.0.0.1'}});
+    server.handler(queryRequestHandler);
```

Comment:
Please inline the handler in this test case.

I find it difficult to understand what's going on in this test case when the query handler is defined elsewhere and the name does not explain what is the handler actually doing.

Alternatively, find a better name for this handler function, e.g. `respondWithParsedQueryObject`.